torch/distributed/checkpoint/_fsspec_filesystem.py

```diff
@@ -50,6 +51,18 @@ def rename(
     def mkdir(self, path: [str, os.PathLike]) -> None:
         self.fs.makedirs(path, exist_ok=True)
 
+    @staticmethod
+    def check(path: Union[str, os.PathLike, None] = None) -> bool:
```

Comment:
fsspec handles local files fine though? What's the motivation here? You can give unprefixed files to fsspect and it will just return a LocalFileSystem, fsspec object?

Reply:
The original motivation is to keep the filesystem reader and writer as the default one as we don't know how fsspec can affect the performance. Also, there was also concern that taking fsspec as the hard dependency. But it is added to `requirements.py`. So it may not be the case anymore.